wasm-client/solver.js

```diff
@@ -412,21 +427,26 @@ module.exports = {
         })
         
         async function handleTimeouts(taskID) {
+            console.log("Handling timeout")
```

Comment:
It would be great if we could use winston instead of console.log here

Reply:
I think it's best to change after all the tests are working again. Would prefer to somehow tune winston so that it prints the extra args to logging method, or at least as object like I had in my test node.

Reply:
Okay, don't disagree. @OR13 any thoughts on customizing winston for this?